gulpfile.coffee

```diff
@@ -79,62 +86,41 @@ onError = (err) ->
 
 
 gulp.task 'script', false, ->
-  gulp.src paths.script
-    .pipe $.plumber(errorHandler: onError)
-    .pipe $.coffee()
-    .pipe $.concat 'script.js'
-    .pipe do $.uglify
-    .pipe $.size {title: 'Minified scripts'}
-    .pipe gulp.dest "#{dir_min}/script"
-
-
-gulp.task 'script_dev', false, ->
-  gulp.src paths.script
+  gulp.src "#{dir_script}/**/*.coffee"
     .pipe $.plumber(errorHandler: onError)
     .pipe do $.sourcemaps.init
     .pipe $.coffee()
-    .pipe $.concat 'script.dev.js'
     .pipe do $.sourcemaps.write
-    .pipe gulp.dest "#{dir_min}/script"
-
-
-gulp.task 'ext', false, ->
-  gulp.src paths.ext
-    .pipe $.plumber(errorHandler: onError)
-    .pipe $.concat 'ext.js'
-    .pipe do $.uglify
-    .pipe $.size {title: 'Minified ext libs'}
-    .pipe gulp.dest "#{dir_min}/script"
-
-
-gulp.task 'ext_dev', false, ->
-  gulp.src paths.ext
-    .pipe $.plumber(errorHandler: onError)
-    .pipe do $.sourcemaps.init
-    .pipe $.concat 'ext.dev.js'
-    .pipe do $.sourcemaps.write
-    .pipe gulp.dest "#{dir_min}/script"
+    .pipe gulp.dest "#{dir_build}/script"
 
 
 gulp.task 'style', false, ->
   gulp.src paths.style
     .pipe $.plumber(errorHandler: onError)
-    .pipe do $.less
-    .pipe $.autoprefixer {cascade: false}
-    .pipe do $.minifyCss
-    .pipe $.size {title: 'Minified styles'}
-    .pipe gulp.dest "#{dir_min}/style"
-
-
-gulp.task 'style_dev', false, ->
-  gulp.src paths.style
-    .pipe $.plumber(errorHandler: onError)
     .pipe do $.sourcemaps.init
     .pipe do $.less
     .pipe do $.sourcemaps.write
-    .pipe $.rename 'style.dev.css'
     .pipe $.autoprefixer {map: true}
-    .pipe gulp.dest "#{dir_min}/style"
+    .pipe gulp.dest "#{dir_build}/style"
+
+
+gulp.task 'inject', false, ->
+  gulp.src("#{dir_main}/templates/bit/script.html")
+    .pipe $.plumber()
+    .pipe $.inject(gulp.src(paths.script),
```

Comment:
Maybe something like that
``` coffeescript
.pipe $.inject(gulp.src("#{dir_build}/script/**/*.js"),
```
And then we do not need to register the script manually. What do you think?